test/widgets/content_test.dart

```diff
@@ -471,6 +471,20 @@ void main() {
       debugNetworkImageHttpClientProvider = null;
     });
 
+   testWidgets('smoke with absolute URL and size(pixelratio * 50)', (tester) async {
+      const avatarUrl = 'https://example/avatar.png';
+      check(await actualUrl(tester, avatarUrl, 50)).isNotNull()
+        .asString.equals(avatarUrl.replaceAll('.png', '-medium.png'));
+      debugNetworkImageHttpClientProvider = null;
+    });
+
+    testWidgets('smoke with relative URL and size(pixelratio * 50)', (tester) async {
+      const avatarUrl = '/avatar.png';
+      check(await actualUrl(tester, avatarUrl, 50))
+        .equals(store.tryResolveUrl(avatarUrl.replaceAll('.png', '-medium.png'))!);
```

Comment:
This is clearer if you just spell out the result:
```suggestion
        .equals(store.tryResolveUrl('/avatar-medium.png')!);
```

The `replaceAll` version is already relying on the details of how this particular `avatarUrl` looks — for example if the URL were `/avatars.pngset/avatar.png`, then this `replaceAll` would rewrite it in two places and get the wrong result.